lib/events.js

```diff
@@ -436,6 +436,15 @@ function listenerCount(type) {
   return 0;
 }
 
+EventEmitter.prototype.eventNames = function eventNames() {
+  if (this._eventsCount > 0) {
+    const events = this._events;
+    return Object.keys(events).concat(
+      Object.getOwnPropertySymbols(events));
```

Comment:
Shouldn't we check if the symbol property is enumerable `events.propertyIsEnumerable(symbol)` and add it to the array only if it is?